3_harmonize/src/clean_wqp_data.R

```diff
@@ -127,7 +141,8 @@ flag_duplicates <- function(wqp_data, duplicate_definition){
     group_by(across(all_of(duplicate_definition))) %>% 
     mutate(n_duplicated = n(),
            flag_duplicated_row = n_duplicated > 1) %>% 
-    ungroup()
+    ungroup() %>%
+    select(-n_duplicated)
```

Comment:
Is this removed and then added to `remove_duplicates()` so that it is removed in case someone drops the `remove_duplicates()` step from the cleaning function?

Reply:
That could be another use case for removing `n_duplicated` within `flag_duplicates()` but the reason I did this was to make `remove_duplicates()` more modular without assuming that someone would have run `flag_duplicates()` beforehand.